testing/test_nodes.py

```diff
@@ -23,6 +23,8 @@
         ("a/b/c::D/d::e", ["", "a", "a/b", "a/b/c::D", "a/b/c::D/d", "a/b/c::D/d::e"]),
         # : alone is not a separator.
         ("a/b::D:e:f::g", ["", "a", "a/b", "a/b::D:e:f", "a/b::D:e:f::g"]),
+        # / not considered if a part of a test name
+        ("a/b::c/d::e[/test]", ["", "a", "a/b::c", "a/b::c/d", "a/b::c/d::e[/test]"]),
```

Comment:
Im under the impression `a/b` is missing as element

But thats for a follow-up as the same detail applies to other nodes as well

Reply:
I see, but I believe that is apart of the functionality of the `iterparentnodeids` function because on line 65 of `src/_pytest/nodes.py` @bluetech put a comment that says https://github.com/pytest-dev/pytest/blob/1fc450658a3733692511beef0f667ae06d45cf12/src/_pytest/nodes.py#L65

Reply:
i mean this may need a reconsideration **much** later, your pr is fine

Reply:
Oh okay. I understand. For this to be merged do I just wait for @bluetech to review it? I am new to the process, sorry.

Reply:
> Oh okay. I understand. For this to be merged do I just wait for @bluetech to review it? I am new to the process, sorry.

Right -- I will try to review it today or tomorrow.